Task2/aga/TicTacToe/TicTacToe/Click.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicTacToe
+{
+    internal class Click
```

Comment:
A 'Click' class that stores turn counter? It looks more like a Game class. Entire code has just 2 classes: 'Click' and 'TheGame'. Click stores turn counter, TheGame does everything else. See 'Single Responsibility Principle' and apply it to the code. To start think about the tic tac toe game in abstract terms, try to describe it and note down nouns and verbs you use to describe it. Nouns are often good candidates for a class and verbs for methods.